CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Distributes functions with `CREATE FUNCTION` command
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
+  `search_path`
+
+* `citus_shards_on_worker` shows all local shards regardless of `search_path`
+
+* Adds `citus.create_object_propagation` GUC to control ddl creation behaviour
+  in transactions
+
+* Sets `create_object_propagation` default to `immediate`
+
+* Uses a fixed `application_name` while connecting to remote nodes
```

Comment:
I guess we later changed to use a dynamic application_name that includes the GPID

Maybe: `The application_name used for internal connections is prefixed with citus_internal`